_src-js/lib/playlist.js

```diff
@@ -64,11 +64,74 @@ export default [{
   id: 'elephantsdream',
   sources: [
     { src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/hls/ed_hd.m3u8', type: 'application/x-mpegurl' },
+  ],
+  duration: 653,
+  textTracks: [
+    {
+      src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/chapters.en.vtt',
+      kind: 'chapters',
+      srclang: 'en',
+      label: 'English'
+    }
+  ],
+  poster: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/poster.png',
+  thumbnail: [
+    {
+      srcset: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/poster.png',
+      type: 'image/png',
+      media: '(min-width: 400px;)'
+    },
+    {
+      src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/poster.png'
+    }
+  ]
+}, {
+  name: "Elephant's Dream (mp4 with sidecar captions, audio description and chapters)",
+  description: 'The film features two men, Proog, who is older and more experienced, and Emo, who is young and nervous, living in a miraculous construction referred to only as "The Machine".',
+  id: 'elephantsdreammp4',
+  sources: [
     { src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/ed_hd.mp4', type: 'video/mp4' },
     { src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/ed_hd.ogg', type: 'video/ogg' }
   ],
   duration: 653,
   textTracks: [
+    {
+      src: '//d2zihajmogu5jn.cloudfront.net/elephantsdream/captions.en.vtt',
+      kind: 'captions',
+      srclang: 'en',
+      label: 'English',
+      default: true
```

Comment:
Let's take out the `default` on this, for the sake of the demo, so that people can turn on the audio description track without having to turn off this captions track first.